### DISCUSSION TITLE
Frontend changes

### DISCUSSION THREAD

**Discussion by marco-qg** [Arquisoft/wiq_es6c#69] [Ideas]
So, as we discussed last meeting, our goal for next week is to create a good interface.
I've been working with https://uiverse.io/Tiagoadag/cuddly-catfish-6 and I think I did a pretty good job.
This is just a example made in question's history but i think we should extend it to the other views too if you like it.
Let me know what you think.

![image](https://github.com/Arquisoft/wiq_es6c/assets/145330494/b54d6add-42d6-4fe5-8289-d48a5a69b6ab)

**Comment by uo288574** [comment]
I think it is a very original design. In my opinion, it is perfect. Maybe, from this, we could make a similar design for the rest of the views.

**Reply by AbelMH1** [reply]
I agree with you. Maybe the hover effect might be better only for selectable objects, but it looks great anyway!

**Reply by marco-qg** [reply]
> I agree with you. Maybe the hover effect might be better only for selectable objects, but it looks great anyway!

I had thought about that, but i think it might be a bit dull without hover effects. However, if most people prefer it to be that dull, there's no problem in changing it. So let's have @alegarman2002 and @MarcosBarrilVillaverde cast the final votes.

**Comment by marco-qg** [comment]
So, after completely wasting my time studying SSI, I 've continued changing the frontend desing.
# Two new options for game modes:
## Option A: Formal
![image](https://github.com/Arquisoft/wiq_es6c/assets/145330494/97113282-85dd-4163-b2b0-0f400e7701fa)

## Option B: Not that formal
![image](https://github.com/Arquisoft/wiq_es6c/assets/145330494/0f17840a-d741-4062-85ad-653a0bfeb905)

## My view
I'm not that sure, but I think I prefer option B. Also, I'm sure @ExarcaFidalgo (and perhaps the rest of the professors whom I dare not mention) will laugh about that.
By the way, @CANCI0 said that they were both bad desings for showing game modes, so i don't know what to think to be honest :(

**Reply by uo288574** [reply]
I hope you did well on the SSI exam. 
As for the design, I think it's a good choice to give it a more casual feel. 
For the menu, @alegarman2002 , @AbelMH1  and I had discussed to show some default game options (such as an intermediate and a difficult game) and then another option in which the player could customize some aspects of the game. 
So I am not able to visualize how to use these designs in the menu. 
Anyway, I think it's a good design for the questions of each game. And, maybe, in the menu use something simpler like the other options you mentioned.

**Reply by AbelMH1** [reply]
I really like option B. However, I would make the font size a bit larger

**Comment by marco-qg** [comment]
## Option C: Glowy
![image](https://github.com/Arquisoft/wiq_es6c/assets/145330494/7316220a-135a-489a-a15f-d5be814bf1db)
I think this is probably the worst,

**Comment by marco-qg** [comment]
## Option D(I hope last one):
![image](https://github.com/Arquisoft/wiq_es6c/assets/145330494/04d2b4ea-72a7-40af-8978-f254ff7b4030)
Same as storeQuestions, makes sense i guess.

**Reply by uo288574** [reply]
Between options C and D, I think this is my favorite.

**Comment by alegarman2002** [comment]
In my opinion the D option is the best. That is because we cas mantain the same styling in all the web-page
